### DISCUSSION TITLE
Not opening docx files correctly since the last update

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2285] [General] [answered]
After your last update which I installed today I find opening a .docx gives me what looks like the file as text rather than opening it with something that understands it is a docx. There was a Google Docs update at the same time, I don't know if Markor delegates editing docx files to Google Docs or not, but there isn't any other editor on my tablet that could do it (unless you've built it into Markor) so I expect it does. Or it did, and now it does something different.
Anyway Google Docs running by itself opens docx files just fine.
I mostly use Markor for md files but, because it has nicer file navigation, I use it for docx too. Using Google Docs directly it always prompts me to save in Google Drive which I don't want. Markor just let's me do what I want.
Anyway, good product, but not so good for me just now.

**Comment by gsantner** [comment]
As far I see it is fixed in the development version, can you please check?

----

Unreleased changes can be tried in the development version called Marder. It can be installed beside Markor and downloaded from [here](https://github.com/gsantner/markor/actions?query=branch%3Amaster++actor%3Agsantner++) (click on a recent change and find the android-apk button there). Start Marder to try, not Markor in this case.

**Comment by ghost** [comment]
I went to the link and downloaded android-apk. It has a file called net.gsantner.markor-v150-2.12.1-flavorAtest-debug.apk in it, which looks like it is Markor rather than Marder. I also looked at the all.zip file and it too had no sign of Marder, just Markor.
Please advise.

**Comment by gsantner** [comment]
flavorAtest = Marder. However there is v2.12.1 (Markor) also in GitHub 
Releases by now

**Comment by ghost** [comment]
I installed v2.12.1 (Mardor) okay but it still has the exact same problem. You can reproduce it if you go into the Sample Files dir and try opening the ACME Quote-word 2007.docx file.

**Comment by gsantner** [comment]
The latest Marder version ("msword") doesn't treat them as textfiles 
anymore now. Thought it was one of the general issues but it's 
openxml/word specific.

**Comment by mnisius** [comment]
@gsantner This also happens for asciidoc files. So view mode is complete broken for these files. I can confirm that the development version fixes the issue. Will there be a release soon? All my notes are in asciidoc ...

**Comment by ghost** [comment, marked as answer]
I just got the latest update (2.12.2) and can confirm it is handling .docx files just fine now.
Thanks for the fix.
